simapp/test_helpers.go

```diff
@@ -440,3 +437,13 @@ type EmptyAppOptions struct{}
 func (ao EmptyAppOptions) Get(o string) interface{} {
 	return nil
 }
+
+// FundAccount is a utility function that funds an account by minting and sending the coins to the address
+// TODO(fdymylja): instead of using the mint module account, which has the permission of minting, create a "faucet" account
```

Comment:
This is for tests, why can't we use the `bank/keeper.MintCoins`?

Reply:
it's commented, mint coins needs a module account with permissions to mint coins. the correct flow to correctly track supply is that a module mints and after it sends the coins to the account.

Reply:
OK, but my question is why doing multiple minting is not good?

Reply:
because it's a mint and send, the PR size would've been 2x, and I think having a utility function it's consistent with we have had so far in simapp (AddAddrsAndCoins etc), and also does not leave other devs lost at what to do when they will need to set balance somewhere without using SetBalance

Reply:
Sorry, I didn't express my question clearly enough.

I know that we are doing mint+send here, and it's correct as you said. In the TODO you wrote that we should use faucet account and send tokens from that faucet account (instead of mint+send). It's not clear to me why faucet is superior to what you have implemented (mint+send).